elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAutoConfiguration.java

```diff
@@ -53,6 +54,7 @@ public Elide initializeElide(EntityDictionary dictionary,
                 .withEntityDictionary(dictionary)
                 .withDefaultMaxPageSize(settings.getMaxPageSize())
                 .withDefaultPageSize(settings.getPageSize())
+                .withJSONApiLinks(new DefaultJSONApiLinks())
```

Comment:
This should be disabled by default in Elide 4.  However, turn it on for testing so the tests below work.